modules/gitops-release-notes-1-2.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-2_{context}"]
+= Release notes for {gitops-title} 1.2
+
+{gitops-title} 1.2 is now available on {product-title} 4.7 and 4.8.
+
+[id="support-matrix-1-2_{context}"]
+== Support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Support matrix
+[cols="1,1",options="header"]
+|===
+| Feature | {gitops-title} 1.2
+| Argo CD
+| GA
+| Argo CD ApplicationSet
+| TP
+| {gitops-title} Application Manager (kam)
+| TP
+| {gitops-title} Service
+| TP
+|===
+
+[id="new-features-1-2_{context}"]
+== New features
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.2:
+
+* If you do not have read or write access to the openshift-gitops namespace, you can now use the `DISABLE_DEFAULT_ARGOCD_INSTANCE` environment variable in the GitOps Operator and set the value to `TRUE` to prevent the default Argo CD instance from starting in the `openshift-gitops` namespace.
+* Resource requests or limits are now configured in Argo CD workloads.  Resource quota is enabled in the `openshift-gitops` namespace. As a result, out-of-band workloads deployed manually in the openshift-gitops namespace must be configured with resource requests or limits and the resource quota may need to be increased.
```

Comment:
should it be `Resource requests and limits are now configured for Argo CD workloads` ? since we are defining both and not just requests or limits?

Reply:
also, I think there's an additional `space` after `workloads.  Resource quota is enabled ...` in L44

Reply:
Two spaces are a period, right?

Reply:
yepp .. it looks fine in the preview !!

Reply:
what is our consensus? 
1. resource requests or limits
2. resource requests and/or limits
3. resource requests and limits
4.

Reply:
@Preeticp

Reply:
@wtam2018 based on @shubhamagarwal19 's inputs the 3rd option: resource requests and limits seems a better option to me.